static/styles/zulip.css

```diff
@@ -2697,3 +2718,16 @@ img.emoji {
         margin-top: 10px;
     }
 }
+
+.high-contrast a {
+    color: hsl(200, 100%, 36%);
+}
+
+[aria-hidden="true"] {
+    visibility: hidden;
+}
```

Comment:
I think this change is really risky; there are lots of things one wants to hide from screen-readers for usability but doesn't want to hide in general, and I feel like this could create a lot of subtle bugs.  Is this change something you saw recommended somewhere?